core/templates/pages/contact-page/contact-page.component.html

```diff
@@ -48,6 +48,10 @@ <h2 [innerHTML]="'I18N_CONTACT_PAGE_OTHER_INQUIRIES_HEADING' | translate"></h2>
     width: 100px;
   }
 
+  ::ng-deep .oppia-contact-mail {
```

Comment:
As mentioned previously, please add a comment here explaining why the ng-deep is necessary, together with a link to the StackOverflow URL brought up earlier. This is important for future developers who might not have an idea of why we are doing this and might accidentally remove it because they don't have context, thus undoing all your work.

Reply:
@seanlip I agree that will definitely help future devs. I have added the comment PTAL. Thanks!